### PR TITLE
5935 fix story spacing

### PR DESCRIPTION
**Pull request by brett-varney**
https://federal-spending-transparency.atlassian.net/browse/DA-5935

You can try to follow the code changes, but it doesn't make much sense to me. It just works (unless it doesn't, just let me know how).

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -268,7 +268,7 @@ $mdGrey: #555555;
 .MuiGrid-item.intro {
   font-size: 2.5rem;
   font-weight: 300;
-  margin-bottom: $margin-30;
+  // margin-bottom: $margin-30;
```
Comment (anjenkin) [inline src/styles/index.scss]: delete?

**Comment by anjenkin** [comment]
There doesn't seem to be enough space between the sections.
Screen Shot 2020-03-25 at 10 58 22 AM

The story section header is visually different than the mocks now.

Screen Shot 2020-03-25 at 10 56 38 AM

**Comment by brett-varney** [comment]
> 
> 
> See comments and screenshots attached.

Made adjustments; how about now? Thanks for being thorough.

**Comment by anjenkin** [comment]
Section header looks good.  Spacing on C & U looks good and federal employee page looks good.  There were a couple spacing issues on homelessness.  Also, the tooltips on the homelessness page are rendering inline.  

Inline tooltip - 

Screen Shot 2020-03-25 at 1 05 46 PM


Extra line and extra spacing - 

Screen Shot 2020-03-25 at 1 02 22 PM

**Comment by brett-varney** [comment]
> 
> 
> Section header looks good. Spacing on C & U looks good and federal employee page looks good. There were a couple spacing issues on homelessness. Also, the tooltips on the homelessness page are rendering inline.
> 
> Inline tooltip -
> 
> Screen Shot 2020-03-25 at 1 05 46 PM
> 
> Extra line and extra spacing -
> 
> Screen Shot 2020-03-25 at 1 02 22 PM

These are also good finds. The problem with the accordion spacing is how the accordion is designed. Collapsed, each item still takes up over 8px, so more items will have more spacing. I'll write that up, but beyond the scope of this ticket.

And the C&U and Homeless pages add different bottom margins to ULs., outside the templates.

**Comment by brett-varney** [comment]
> 
> 
> Section header looks good. Spacing on C & U looks good and federal employee page looks good. There were a couple spacing issues on homelessness. Also, the tooltips on the homelessness page are rendering inline.
> 
> Inline tooltip -
> 
> Screen Shot 2020-03-25 at 1 05 46 PM
> 
> Extra line and extra spacing -
> 
> Screen Shot 2020-03-25 at 1 02 22 PM

As for tool tips, well, they are each completely different code bases, so I'd rather reconcile them and remove the duplicate, but I'll see what I can do easier...

**Comment by brett-varney** [comment]
> As for tool tips, well, they are each completely different code bases, so I'd rather reconcile them and remove the duplicate, but I'll see what I can do easier...

Fixed 'em. The base style was never imported into the viz's.

File: src/components/visualizations/homelessness-analysis/mapviz/mapviz.jsx [hunk 2]
```diff
@@ -236,8 +236,8 @@ export default function Mapviz(props) {
   });
 
 return (<div className={props.display ? '' : 'hidden'}>
-      <div id="tooltipMapViz" className="tooltip-module" />
-      <div className="viz-container homeless-analysis">
+      <div id="tooltipMapViz" className={tipStyles.tooltipModule} />
+      <div className={`${styles.vizContainer} ${styles.homelessAnalysis}`}>
```
Comment (brett-varney) [inline src/components/visualizations/homelessness-analysis/mapviz/mapviz.jsx]: here

File: src/page-sections/homelessness-analysis/federal-programs.jsx [hunk 3]
```diff
@@ -900,22 +901,22 @@ export default function FederalPrograms(props) {
...
         </Grid>
       </Grid>
       <Grid container>
         <Grid item xs={12} md={6}>
-          <div id='container2_3' className='homelessness-container2 homelessness-container2--left' />
+          <div id='container2_3' className={`${styles.homelessnessContainer2} ${styles.homelessnessContainer2Left}`} />
         </Grid>
         <Grid item xs={12} md={6}>
-          <div id='container2_4' className='homelessness-container2 homelessness-container2--right' />
+          <div id='container2_4' className={`${styles.homelessnessContainer2 } ${styles.homelessnessContainer2Right}`} />
```
Comment (brett-varney) [inline src/page-sections/homelessness-analysis/federal-programs.jsx]: and here

**Comment by anjenkin** [comment]
Looks good!

**Pull request merged**